hedera-node/src/test/java/com/hedera/services/state/exports/SignedStateBalancesExporterTest.java

```diff
@@ -626,17 +622,39 @@ private String expectedExportDir() {
 	public void initsAsExpected() {
 		// expect:
 		assertEquals(ledgerFloat, subject.expectedFloat);
-		assertEquals(SignedStateBalancesExporter.NEVER, subject.periodEnd);
 	}
 
 	@Test
 	public void exportsWhenPeriodSecsHaveElapsed() {
-		assertFalse(subject.isTimeToExport(now));
-		assertEquals(now.plusSeconds(dynamicProperties.balancesExportPeriodSecs()), subject.periodEnd);
+		// test the export won't happen inside interval
+		subject = new SignedStateBalancesExporter(properties, signer, dynamicProperties);
+		Instant startTime = Instant.parse("2021-03-11T10:01:32.0Z");
+		assertFalse(subject.isTimeToExport(startTime));
+		assertEquals(startTime, subject.periodBegin);
+		assertFalse(subject.isTimeToExport(startTime.plusSeconds(1)));
+		assertEquals(startTime.plusSeconds(1), subject.periodBegin);
+		assertFalse(subject.isTimeToExport(startTime.plusSeconds(11)));
+		assertEquals(startTime.plusSeconds(11), subject.periodBegin);
```

Comment:
This portion of the test is redundantly covered by the rest of the test. Plus it is using magic numbers, not the `dynamicProperties.balancesExportPeriodSecs()` that can clear show whether a new timestamp will trigger a balance export or not.

Reply:
Yeah. This portion is redundant. Removed.